sdcm/utils/common.py

```diff
@@ -1132,14 +1132,14 @@ def filter_k8s_clusters_by_tags(tags_dict: dict, clusters: list[
 @lru_cache
 def get_scylla_ami_versions(region_name: str, arch: AwsArchType = 'x86_64', version: str = None) -> list[EC2Image]:
```

Comment:
it's used by `get_latest_scylla_ami_release` which is not used anywhere, maybe remove this function along so it's not distracting?

Reply:
```
$ git grep get_scylla_ami_versions
sdcm/sct_config.py:    get_scylla_ami_versions,
sdcm/sct_config.py:                            ami = get_scylla_ami_versions(version=scylla_version, region_name=region, arch=aws_arch)[0]
sdcm/sct_config.py:                            ami = get_scylla_ami_versions(version=oracle_scylla_version,
sdcm/utils/common.py:def get_scylla_ami_versions(region_name: str, arch: AwsArchType = 'x86_64', version: str = None) -> list[EC2Image]:
sdcm/utils/common.py:    for ami in get_scylla_ami_versions(region_name=region):
sdcm/utils/common.py:            for ami in get_scylla_ami_versions(region_name=region_name, arch=arch, version=version)]
unit_tests/test_scylla_yaml_builders.py:            with patch("sdcm.sct_config.get_scylla_ami_versions", return_value=[self.get_scylla_ami_version_output]), \

```

Reply:
I meant `get_latest_scylla_ami_release` to be removed